lambda/main.go

```diff
@@ -98,13 +99,21 @@ func Handler(ctx context.Context, evt json.RawMessage) (string, error) {
 		timeout = "15"
 	}
 
+	if maxIdleConns == "" {
+		maxIdleConns = "100" // Default to 100 in line with http.DefaultTransport
+	}
+
```

Comment:
For neatness, I'd move this lower immediately above the code that then parses `maxIdleConns`

Reply:
@DrJosh9000 great point, this kind of irked me so I extracted a function and added a basic test. Hopefully that looks better 🙏🏻